ribbon-loadbalancer/src/main/java/com/netflix/client/PrimeConnections.java

```diff
@@ -126,26 +125,21 @@ private PrimeConnections() {
     }
 
     public PrimeConnections(String name, IClientConfig niwsClientConfig) {
-        int maxRetriesPerServerPrimeConnection = Integer.valueOf(DefaultClientConfigImpl.DEFAULT_MAX_RETRIES_PER_SERVER_PRIME_CONNECTION);
-        long maxTotalTimeToPrimeConnections = Long.valueOf(DefaultClientConfigImpl.DEFAULT_MAX_TOTAL_TIME_TO_PRIME_CONNECTIONS);
-        String primeConnectionsURI = DefaultClientConfigImpl.DEFAULT_PRIME_CONNECTIONS_URI;  
-        String className = DefaultClientConfigImpl.DEFAULT_PRIME_CONNECTIONS_CLASS;
+        int maxRetriesPerServerPrimeConnection = CommonClientConfigKey.MaxRetriesPerServerPrimeConnection.getDefaultValue();
+        int maxTotalTimeToPrimeConnections =  CommonClientConfigKey.MaxTotalTimeToPrimeConnections.getDefaultValue();
```

Comment:
This one is `int` and `long` interchangeably, based on the usage I'd stick to `int`.

Reply:
It's really an int everywhere.  It's consistent now.